### PR TITLE
[docs] Various polish / docs / spelling mistake stuff.

### PR DESCRIPTION
**Pull request by gburtini**
Why
===

I had a long layover in Vancouver. Faris says we're doing docs. This is probably a bit better.

### CODE REVIEW

**Review by masad-frost — APPROVED** [review]
haha, nice

File: src/client.ts [hunk 1]
```diff
@@ -1280,12 +1274,12 @@ export class Client<Ctx = null> {
...
-     * the connection if we don't get READY from the container
-     * within the specified timeout.
+     * If the container stops responding for `timeout` period, we will throw.
+     * The container is expected to regularly send status updates if it is still
+     * working and healthy.
      *
-     * Every time we get a message we reset the connection timeout
-     * this is because it signifies that the connection will eventually work.
+     * Every time we get a message we reset the connection timeout this is because
+     * it signifies that the connection should eventually work.
```
Comment (masad-frost) [inline src/client.ts:1282]: will->should :grimacing: lol

Comment (gburtini) [reply src/client.ts:1282]: _will_ wasn't correct, especially now that we're blindly sending status update events every 500ms.

Comment (gburtini) [reply src/client.ts:1282]: it pretty much just means "this conman is still alive", not "this connection is still making progress."

**Pull request merged**